templates/home.html

```diff
@@ -5,6 +5,9 @@
   <main role="main" class="homepage">
     <div class="container">
 <!--       Home page content here -->
+      {{included.contents|safe}}
+      {{included.testKey|safe}}
+      {{included.sourcefile|safe}}
```

Comment:
@robloach please review : https://github.com/treygriffith/metalsmith-include/pull/6

On June 17, 2015 9:15:24 PM EDT, Rob Loach <notifications@github.com> wrote:
>> @@ -5,6 +5,9 @@
>>    <main role="main" class="homepage">
>>      <div class="container">
>>  <!--       Home page content here -->
>> +      {{included.contents|safe}}
>> +      {{included.testKey|safe}}
>> +      {{included.sourcefile|safe}}
>
>Looked at the fork... Perhaps rather than having include use Markdown
>for everything, we might be able to use a `markdown` filter...
>
>https://github.com/paularmstrong/swig-extras/blob/master/lib/filters/markdown.js
>```json
>{
>  "metalsmith-swig-helpers": {
>    "filters": {
>      "markdown": "swig-extras/lib/filters/markdown"
>    }
>  }
>}
>```
>``` html
>{{included.contents|safe|markdown}}
>{{included.testKey|safe|markdown}}
>{{included.sourcefile|safe|markdown}}
>```
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/kalamuna/kalastatic/pull/110/files#r32692255

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.